tests/migration_test.go

```diff
@@ -2954,6 +2954,60 @@ var _ = Describe("[Serial][rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][leve
 			Expect(imageIDs).To(HaveKeyWithValue(container.Name, digest), "expected image:%s for container %s to be the same like on the source pod but got %s", container.Image, container.Name, imageIDs[container.Name])
 		}
 	})
+
+	Context("CLI output", func() {
```

Comment:
Wouldn't this be better fit in `kubectl_test.go`?

Reply:
@xpivarc Hi. Thank you for the notice! I was also thinking to put the test there but wasn't completely sure where it fits most. The best solution is probably to have a dedicated CLI section of e2e, but thats for the follow up.

Reply:
@xpivarc Done